### ISSUE TITLE
Inductive probes with horn duct conflict

### ISSUE THREAD

**Issue by M1dn1ghtN1nj4** [pkucmus/EVA#16]
For the setup I'm looking to put together, there seems to be a problem with the both the 8mm and 12mm inductive bed probes with the horn duct.  Or could it be that I'm somehow not assembling this right.  The photos show it like it fits, but I can't find a combination that works.

![Untitled](https://user-images.githubusercontent.com/40809511/82941468-65b30b00-9f64-11ea-8fac-7052190b2c7e.jpg)

**Comment by pkucmus** [comment]
You are probably looking for the "_right" variants of the probes.
![image](https://user-images.githubusercontent.com/6347418/82990937-39979880-9ffd-11ea-9cd8-9da4bffdfb08.png)
![image](https://user-images.githubusercontent.com/6347418/82991018-53d17680-9ffd-11ea-9ced-b07f38399ee5.png)
All probe holder variants are here: https://github.com/pkucmus/EVA/tree/master/stl/Probe%20Mounts

Please let me know if that helps.

**Comment by pkucmus** [comment]
Any updates?

**Comment by M1dn1ghtN1nj4** [comment]
Yep, that works.  I picked the left side one because it brings the probe more in line with the nozzle.  It's how I've designed all of my printers.  The probe and nozzle are perfectly in line in the Y axis.  Just a bit OCD on my part.  I also had to shorten up the horn duct a little.  The output was about 4mm to 5mm ahead of the nozzle tip.
![Untitled](https://user-images.githubusercontent.com/40809511/83305861-816e1980-a1cf-11ea-8968-9fd232e02332.jpg)
